obspy/core/tests/test_trace.py

```diff
@@ -1485,6 +1485,22 @@ def test_split(self):
         self.assertFalse(isinstance(st[0].data, np.ma.masked_array))
         self.assertFalse(isinstance(st[1].data, np.ma.masked_array))
 
+    def test_split_empty_masked_array(self):
+        """
+        Test split method with a masked array without any data.
+        """
+        tr1 = Trace(data=np.arange(1000))
+        tr2 = Trace(data=np.arange(1000, 2000))
+        trace = tr1 + tr2
```

Comment:
It would IMHO be clearer to directly initialize a Trace with a fully masked array, e.g.

```python
tr = Trace(data=np.ma.masked_all(100))
```